ui/push-health/ClassificationGroup.jsx

```diff
@@ -90,15 +90,18 @@ class ClassificationGroup extends React.PureComponent {
       hasRetriggerAll,
       notify,
       currentRepo,
-      searchStr,
```

Comment:
Filtering by `searchStr` now happens in `Health.jsx` rather than down here.  That way all filtering happens at once, and we can control it better.